$NetBSD: patch-ab,v 1.3 2007/02/15 14:28:40 rillig Exp $

Tags: for-upstream

- Assume we are never crosscompiling.
- Don't know why the DB version "4" is explicitly needed.

--- configure.orig	2006-10-29 19:38:07.000000000 +0100
+++ configure	2007-01-12 04:49:50.000000000 +0100
@@ -1906,7 +1906,7 @@ test -n "$target_alias" &&
     NONENONEs,x,x, &&
   program_prefix=${target_alias}-
 
-if test "$build" != "$host" ; then
+if false; then
     CC=$host_alias-gcc
     CXX=$host_alias-c++
     AR=$host_alias-ar
@@ -5321,7 +5321,7 @@ echo $ECHO_N "checking for Berkeley DB..
 echo $ECHO_N "checking for Berkeley DB >= $minversion... $ECHO_C" >&6; }
   fi
 
-  for version in "" 5.0 4.9 4.8 4.7 4.6 4.5 4.4 4.3 4.2 4.1 4.0 3.6 3.5 3.4 3.3 3.2 3.1 ; do
+  for version in "" 5.0 4.9 4.8 4.7 4.6 4.5 4.4 4.3 4.2 4.1 4.0   4   3.6 3.5 3.4 3.3 3.2 3.1 ; do
 
     if test -z $version ; then
         db_lib="-ldb"
